tests/test_stats_beta_logpdf.py

```diff
@@ -7,41 +7,38 @@
 
 sess = tf.Session()
 
-
 def _assert_eq(val_ed, val_true):
     with sess.as_default():
-        # NOTE: since Tensorflow has no log_beta function, the values here are
+        # NOTE: since Tensorflow has no special functions, the values here are
         # only an approximation
         assert np.allclose(val_ed.eval(), val_true, atol=1e-4)
 
-
-def _test_logpdf_scalar(scalar, a=.5, b=.5):
-    x = tf.constant(scalar)
-    val_true = stats.beta.logpdf(scalar, a, b)
-    _assert_eq(beta.logpdf(x, tf.constant(a), tf.constant(b)), val_true)
-    _assert_eq(beta.logpdf(x, tf.constant([a]), tf.constant(b)), val_true)
-    _assert_eq(beta.logpdf(x, tf.constant(a), tf.constant([b])), val_true)
-    _assert_eq(beta.logpdf(x, tf.constant([a]), tf.constant([b])), val_true)
-
+def _test_logpdf_scalar(x, a=0.5, b=0.5):
+    xtf = tf.constant(x)
+    val_true = stats.beta.logpdf(x, a, b)
+    _assert_eq(beta.logpdf(xtf, tf.constant(a), tf.constant(b)), val_true)
+    _assert_eq(beta.logpdf(xtf, tf.constant([a]), tf.constant(b)), val_true)
+    _assert_eq(beta.logpdf(xtf, tf.constant(a), tf.constant([b])), val_true)
+    _assert_eq(beta.logpdf(xtf, tf.constant([a]), tf.constant([b])), val_true)
 
 def test_logpdf_scalar():
     _test_logpdf_scalar(0.3)
     _test_logpdf_scalar(0.7)
 
-    _test_logpdf_scalar(.3, a=1., b=1.)
-    _test_logpdf_scalar(.7, a=1., b=1.)
-
-    _test_logpdf_scalar(.3, a=.5, b=5.)
-    _test_logpdf_scalar(.7, a=.5, b=5.)
+    _test_logpdf_scalar(0.3, a=1.0, b=1.0)
+    _test_logpdf_scalar(0.7, a=1.0, b=1.0)
 
-    _test_logpdf_scalar(.3, a=5., b=.5)
-    _test_logpdf_scalar(.7, a=5., b=.5)
+    _test_logpdf_scalar(0.3, a=0.5, b=5.0)
+    _test_logpdf_scalar(0.7, a=0.5, b=5.0)
```

Comment:
Yeah we test both cases in the script, the diff just looks like the arguments are switched.